common/src/main/java/org/opencds/cqf/common/helpers/FhirContextHelper.java

```diff
@@ -0,0 +1,16 @@
+package org.opencds.cqf.common.helpers;
+
+import ca.uhn.fhir.context.FhirContext;
+
+public class FhirContextHelper {
```

Comment:
I like the idea of caching, and I know I told you to pull this in, but upon review I think we don't need it. Let's delete this class for now.